Sources/XcodeProj/Project/XCUserData.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+import PathKit
+import AEXML
+
+public final class XCUserData: Equatable, Writable {
+    // MARK: - Attributes
+
+    /// User name
+    public var userName: String
+
+    /// User data schemes.
+    public var schemes: [XCScheme]
+
+    /// Metdata for schemes
+    public var schemeManagement: XCSchemeManagement?
+
+    /// User data breakpoints.
+    public var breakpoints: XCBreakpointList?
+
+    // MARK: - Init
+
+    /// Initializes the shared data with its properties.
+    ///
+    /// - Parameters:
+    ///   - userName: User name
+    ///   - schemes: User data schemes.
+    ///   - breakpoints: User data breakpoints.
+    ///   - schemeManagement: Metdata for schemes
+    public init(userName: String,
+                schemes: [XCScheme],
+                breakpoints: XCBreakpointList? = nil,
+                schemeManagement: XCSchemeManagement? = nil) {
+        self.userName = userName
+        self.schemes = schemes
+        self.breakpoints = breakpoints
+        self.schemeManagement = schemeManagement
+    }
+
+    /// Initializes the XCUserData reading the content from the disk.
+    ///
+    /// - Parameter path: path where the .xcuserdatad is.
+    public init(path: Path) throws {
+        if !path.exists {
+            throw XCUserDataError.notFound(path: path)
+        }
+        userName = path.lastComponentWithoutExtension
+
+        let schemesPath = XCScheme.schemesPath(path)
+        schemes = schemesPath
+            .glob("*.xcscheme")
+            .compactMap { try? XCScheme(path: $0) }
+        schemeManagement = try? XCSchemeManagement(path: XCSchemeManagement.path(schemesPath))
+
+        breakpoints = try? XCBreakpointList(path: XCBreakpointList.path(XCDebugger.path(path)))
```

Comment:
Why do we want to ignore the thrown errors here?

Reply:
I would  say we shouldn't ignore the error's here, at least print / log them. But we're not doing that anywhere in similar files, see for example the same line in `XCSharedData`: https://github.com/tuist/XcodeProj/blob/f762375394650ef50d6ed819e811218fddebc844/Sources/XcodeProj/Project/XCSharedData.swift#L43

Reply:
@kwridan do you have any context why we'd want to ignore these erros? But if this is what we're doing elsewhere, I don't mind shipping as-is

Reply:
Keeping the behaviour consistent is preferable 👍 

Looking closely at the code, we may not be able to safely change this without changing the APIs / behaviour. Some of those nested elements are optional, for example you can have an user data folder without breakpoints or schemes - and the nested elements currently throw if the path doesn't exist (which isn't ideal :/) - resulting in making the API unusable unless you have a project with all the sub elements present.